docs/reference/projectyml.rst

```diff
@@ -54,6 +54,63 @@ The code snippet below provides an example of a complete ``project.yml`` file:
       index_urls:
         - https://pypi.org/simple/
 
+
+.. _specify_location_pip:
+
+
+Specify locations from where V2 modules will be installed
+---------------------------------------------------------
+This section explains how to configure your project in order to download v2 modules from any python package repository.
+By default, a project created using the :ref:`project-creation-guide` is configured to install packages from ``https://pypi.org/simple/``. There are multiple ways
+to change this behaviour.
+
+Using pip config file
+"""""""""""""""""""""
+
+By setting the ``use_config_file`` option of the pip section to ``True``, the project will use the pip config files.
+
+.. code-block:: yaml
+
+    pip:
+      use_config_file: True
+
+To specify the url of a pip repository, add the following to the pip config file:
```

Comment:
If you show the content of the file, shouldn't you also specify the path to that file? I know there is a link to the full doc below.  But as a user, I think I would be a bit frustrated to see the simple step that needs to be done in here, and actually have to click one more link and search through a different doc to find the place where that change needs to be applied.

Reply:
do you mean the location of the pip config file? because if I understand correctly it could be located at multiple places.

Reply:
That is what I meant yes, I ever used it so I don't know how it works.  Which also means that I wouldn't know what to do for this step.

Reply:
I guess there is a user level pip config file? Maybe you can advice to edit this one?

Reply:
Ok i'll do that :)